src/components/Accordion.js

```diff
@@ -85,9 +69,9 @@ const allMenuItemsAction = button => {
 const collpasePanelUpdate = (isMenuOpen, menuItem) => {
   menuItem.className = menuState(isMenuOpen ? "open" : "close");
   menuItem.setAttribute("aria-expanded", isMenuOpen);
-  menuItem.closest(
-    ".dg_accordion__collapsible"
-  ).className = `dg_accordion__collapsible ${
+  var element = menuItem.closest(".dg_accordion__collapsible");
```

Comment:
this could probably use a better name like `panelElm` or something along those lines but not blocking.